src/data/Storage.ts

```diff
@@ -54,133 +54,147 @@ export const storage = {
 
 // NOTE: Predefined templates for test purpose
 //       for use just uncomment one of the following lines:
-// _emptyArrayOfCycles().catch((err) => console.error(err));
-// _arrayOfCyclesWithSingleCycle().catch((err) => console.error(err));
-// _arrayOfCyclesWithSingleCycleWithDelay().catch((err) => console.error(err));
-// _halfFilledArrayOfCycles().catch((err) => console.error(err));
-// _fullyFilledArrayOfCycles().catch((err) => console.error(err));
-// _fullyFilledArrayOfCyclesWithDelay().catch((err) => console.error(err));
+
```

Comment:
Why did you remove `emptyArray` template? If you decided to replace this function with `countOfCycles` parameter then I dont agree with that. Previously we had just one function which makes cycles array empty, but now we have multiple function with the same parameter which also can make array empty.

Reply:
This function hasn't been removed. I forgot to leave her challenge in the comments. 
Fixed.